actioncable/app/javascript/action_cable/connection.js

```diff
@@ -134,6 +141,10 @@ Connection.prototype.events = {
         return this.monitor.recordPing()
       case message_types.confirmation:
         this.subscriptions.confirmSubscription(identifier)
+        if (this.reconnectAttempted) {
+          this.reconnectAttempted = false
+          this.subscriptions.notify(identifier, "reconnected")
+        }
         return this.subscriptions.notify(identifier, "connected")
```

Comment:
I should probably iterate over subscriptions with `this.subscriptions.findAll(identifier)` to notify subscribers.

Reply:
Or maybe we should allow `this.subscriptions.notify(identifier, callbackName)` to accept an array of callback names instead of just a string.